lib/nameToColor.js

```diff
@@ -8,9 +8,10 @@ const HEX = {
 
 const RGBA = {
   black: 'rgba(0, 0, 0, 0.75)',
-  red: 'rgba(219, 69, 0, 0.8)',
-  orange: 'rgba(245, 151, 0, 0.8)',
-  yellow: 'rgba(242, 202, 0, 0.7)',
+  blacklight: 'rgba(0, 0, 0, 0.5)',
```

Comment:
Why do we have black and blacklight? There is set of `propTypes` validators at the bottom of a few files that will need to be updated to be aware of this additional property as well.